src/inference/inference.ts

```diff
@@ -72,10 +71,10 @@ export class Inference {
    *  const rerankingModel = "bge-reranker-v2-m3";
    *  const myQuery = "What are some good Turkey dishes for Thanksgiving?";
    *  const myDocuments = [
-   *  { text: "I love turkey sandwiches with pastrami" },
-   *  { text: "A lemon brined Turkey with apple sausage stuffing is a classic Thanksgiving main" },
-   *  { text: "My favorite Thanksgiving dish is pumpkin pie" },
-   *  { text: "Turkey is a great source of protein" },
+   *  { myField: "I love turkey sandwiches with pastrami" },
+   *  { myField: "A lemon brined Turkey with apple sausage stuffing is a classic Thanksgiving main" },
+   *  { myField: "My favorite Thanksgiving dish is pumpkin pie" },
+   *  { myField: "Turkey is a great source of protein" },
```

Comment:
Should this still be `text` because it doesn't look like you are passing `rankFields`